.github/workflows/scripts/coverage.sh

```diff
@@ -5,11 +5,9 @@ set -e
 rm -rf coverage
 mkdir coverage
 
-NIGHTLY="+nightly-2022-06-30"
-
 # Run tests with profiling instrumentation
 echo "Running instrumented unit tests..."
-RUSTFLAGS="-Zinstrument-coverage" LLVM_PROFILE_FILE="identity-%m.profraw" cargo $NIGHTLY test --tests --all --all-features
+RUSTFLAGS="-Zinstrument-coverage" LLVM_PROFILE_FILE="identity-%m.profraw" cargo test --tests --all --all-features
```

Comment:
The `-Z` flag is used for unstable features and although this feature is now available with the stable `-C` flag, I suggest we remove code coverage reports from CI here and start using the now frequently recommended [`cargo-llvm-cov` crate](https://crates.io/crates/cargo-llvm-cov/0.5.0) for this in a later PR.